rental/views.py

```diff
@@ -94,27 +112,101 @@ def hardware_location(request):
 
   return render(request,'hardware_location.html', {'inventories': inventories, 'free_inventories': free_inventories})
 
-def rent_device(request):
+def reserve_device(request):
+  #--------variables from the request
```

Comment:
Need to update this path, make sure the user is authenticated. Because otherwise anyone can hit this path and reserve